src/emulator/gui/src/main_menubar.cpp

```diff
@@ -52,6 +52,15 @@ static void draw_config_menu(HostState &host) {
             }
             ImGui::EndMenu();
         }
+        if (ImGui::BeginMenu("Gui State")) {
+            if (ImGui::MenuItem("Hide Gui", nullptr, !host.cfg.gui_state)) {
```

Comment:
Use a checkbox instead of two different voices.

Reply:
ah yes

Reply:
Done :)